global/meta/models.yml

```diff
@@ -3187,7 +3200,12 @@ chat_group:
     type: number
     default: 10000
     restriction_mode: A
-
+  
+  chat_message_ids:
+    type: relation-list
+    to: chat_message/chat_group_id
+    equal_fields: meeting_id
+    restriction_mode: A
```

Comment:
This should probably also be `on_delete: CASCADE`, since a message cannot exists without a group

Reply:
I added the on_delete to `chat_group/chat_message_ids`.